specs/templates/custom-component.md

```diff
@@ -99,6 +101,8 @@
 
 ### Accessibility
 
+*[Spright - Describe plan, but not necessary to implement initially]*
```

Comment:
like the others, don't think we need a spright specific note

Reply:
I added all those notes based on the task in the issue that says, "Update spec instructions and templates to consider Spright (e.g. annotate existing templates to say which sections are skippable for Spright contributions)". But I guess we really don't want to.